pkg/image/filefmt_test.go

```diff
@@ -60,6 +60,14 @@ var _ = Describe("File format tests", func() {
 			fields{"gz", []byte{0x1F, 0x8B}, 0, 0, 0},
 			[]byte{'Q', 'F', 'I', 0xfb},
 			false),
+		table.Entry("match vmdk",
```

Comment:
what about vhd & vhdx?

Reply:
ah didn't notice that the original pr added vmdk and vdi and didn't add my new vhd and vhdx here

Reply:
Added those